### PR TITLE
Pallet updater

### PR DESCRIPTION
**Pull request by Chubek**
- Added `pallet_updater` to `runtime/cargo.toml` of `/bin`. This includes the system dependencies, std, benchmark e tc.
- Imported `pallet_updater` for us in `runtime/src/lib.rs`
- Added `pallet_updater` config impl in the aforementioned file
- Added `pallet_updater` to `construct_runtime` macro.

### CODE REVIEW

File: bin/node/runtime/Cargo.toml [hunk 1]
```diff
@@ -227,4 +231,6 @@ try-runtime = [
 	"pallet-society/try-runtime",
 	"pallet-recovery/try-runtime",
 	"pallet-vesting/try-runtime",
+	"pallet-updater/try-runtime",
+
```
Comment (charmitro) [inline bin/node/runtime/Cargo.toml:235]: Unnecessary line

File: bin/node/runtime/Cargo.toml [hunk 2]
```diff
@@ -90,6 +90,8 @@ pallet-transaction-payment = { version = "4.0.0-dev", default-features = false,
 pallet-transaction-payment-rpc-runtime-api = { version = "4.0.0-dev", default-features = false, path = "../../../frame/transaction-payment/rpc/runtime-api/" }
 pallet-transaction-storage = { version = "4.0.0-dev", default-features = false, path = "../../../frame/transaction-storage" }
 pallet-vesting = { version = "4.0.0-dev", default-features = false, path = "../../../frame/vesting" }
+pallet-updater = { version = "4.0.0-dev", default-features = false, path = "../pallets/pallet-updater" }
+
```
Comment (charmitro) [inline bin/node/runtime/Cargo.toml:94]: Unnecessary line

File: bin/node/runtime/src/lib.rs [hunk 3]
```diff
@@ -82,6 +82,7 @@ pub use pallet_staking::StakerStatus;
 #[cfg(any(feature = "std", test))]
 pub use sp_runtime::BuildStorage;
 
+
```
Comment (charmitro) [inline bin/node/runtime/src/lib.rs:85]: Unnecessary line

File: bin/node/runtime/src/lib.rs [hunk 4]
```diff
@@ -99,6 +100,9 @@ use constants::time_dev::*;
 
 use sp_runtime::generic::Era;
 
+
```
Comment (charmitro) [inline bin/node/runtime/src/lib.rs:103]: Unnecessary line

File: bin/node/runtime/src/lib.rs [hunk 5]
```diff
@@ -1036,6 +1045,8 @@ construct_runtime!(
 		Lottery: pallet_lottery::{Pallet, Call, Storage, Event<T>},
 		TransactionStorage: pallet_transaction_storage::{Pallet, Call, Storage, Inherent, Config<T>, Event<T>},
 		BagsList: pallet_bags_list::{Pallet, Call, Storage, Event<T>},
+		Updater: pallet_updater::{Pallet, Call, Config<T>, Storage, Event<T>},
+
```
Comment (charmitro) [inline bin/node/runtime/src/lib.rs:1049]: Unnecessary line

File: bin/node/runtime/src/lib.rs [hunk 6]
```diff
@@ -1368,6 +1379,7 @@ impl_runtime_apis! {
 			list_benchmark!(list, extra, pallet_treasury, Treasury);
 			list_benchmark!(list, extra, pallet_utility, Utility);
 			list_benchmark!(list, extra, pallet_vesting, Vesting);
+
```
Comment (charmitro) [inline bin/node/runtime/src/lib.rs:1382]: Unnecessary line

**Review by charmitro — CHANGES_REQUESTED** [review]
Please run `cargo fmt` before pushing changes.

**Comment by charmitro** [comment]
@zycon91 @Chubek Please make sure this work as intended and move it out of DRAFT mode once you're sure about it.

**Comment by Chubek** [comment]
I am marking this to review. Me and @zycon91 ran and tested it. Fotis wrote the fixes which we can see in pallet-updater-fixes. Time to set it for review.

**Review by charmitro — APPROVED** [review]

**Pull request merged**